.golangci.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I don think this file should be checked in, should it?

Reply:
it should be added 
https://github.com/rancher/k3s/blob/master/.golangci.json
because it's used in golint-ci in drone.